charts/shoot-core/components/charts/node-problem-detector/templates/daemonset.yaml

```diff
@@ -2,103 +2,6 @@ apiVersion: {{ include "daemonsetversion" . }}
-      serviceAccountName: node-problem-detector
-      hostNetwork: {{ .Values.hostNetwork }}
-      terminationGracePeriodSeconds: 30
-      {{- if .Values.priorityClassName }}
-      priorityClassName: {{ .Values.priorityClassName | quote }}
-      {{- end }}
-      containers:
-        - name: {{ .Chart.Name }}
-          image:  {{ index .Values.images "node-problem-detector" }}
-          imagePullPolicy: {{ .Values.images.pullPolicy | default "IfNotPresent" | quote }}
-          command:
-            - "/bin/sh"
-            - "-c"
-            - "exec /node-problem-detector --logtostderr --config.system-log-monitor={{- range $index, $monitor := .Values.settings.log_monitors }}{{if ne $index 0}},{{end}}{{ $monitor }}{{- end }} .. {{- if .Values.settings.custom_plugin_monitors }} --config.custom-plugin-monitor={{- range $index, $monitor := .Values.settings.custom_plugin_monitors }}{{if ne $index 0}},{{end}}{{ $monitor }}{{- end }} {{- end }} .. {{- if .Values.settings.system_stats_monitor }} --config.system-stats-monitor={{- range $index, $monitor := .Values.settings.system_stats_monitor }}{{if ne $index 0}},{{end}}{{ $monitor }}{{- end }} {{- end }} --prometheus-address={{ .Values.settings.prometheus_address }} --prometheus-port={{ .Values.settings.prometheus_port }}"
-          securityContext:
-            privileged: true
-          env:
-            - name: NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
-{{- if .Values.env }}
-{{ toYaml .Values.env | indent 12 }}
-{{- end }}
-          volumeMounts:
-            - name: log
-              mountPath: /var/log
-            - name: localtime
-              mountPath: /etc/localtime
-              readOnly: true
-            - name: kmsg
-              mountPath: /dev/kmsg
-              readOnly: true
-          ports:
-            - containerPort: {{ .Values.settings.prometheus_port }}
-              name: exporter
-          resources:
-            requests:
-              cpu: 20m
-              memory: 20Mi
-            limits:
-            {{- if .Values.global.vpaEnabled }}
-              cpu: 80m
-              memory: 80Mi
-            {{- else }}
-              cpu: 200m
-              memory: 100Mi
-            {{- end }}
-    {{- with .Values.affinity }}
-      affinity:
-{{ toYaml . | indent 8 }}
-    {{- end }}
-    {{- with .Values.tolerations }}
-      tolerations:
-{{ toYaml . | indent 8 }}
-    {{- end }}
-{{- if .Values.nodeSelector }}
-      nodeSelector:
-{{ toYaml .Values.nodeSelector | indent 8 }}
-{{- end }}
-      volumes:
-        - name: log
-          hostPath:
-            path: {{ .Values.hostpath.logdir }}
-        - name: localtime
-          hostPath:
-            path: /etc/localtime
-            type: "FileOrCreate"
-        - name: kmsg
-          hostPath:
-            path: /dev/kmsg
+  annotations:
+     resources.gardener.cloud/mode: Ignore
```

Comment:
Please leave a new line at the end of the yamls.

Reply:
I have seen yamls in the master with no new lines at the end. Is it really needed or it is just a good practice ?